package.json

```diff
@@ -2,6 +2,12 @@
   "name": "mahjong-kings",
   "version": "0.1.0",
   "private": true,
+  "jest": {
+    "collectCoverageFrom": [
+      "src/modules/**.{js,jsx,ts,tsx}",
+      "!<rootDir>/node_modules/"
```

Comment:
collect test coverage from `node_modules`?

Reply:
! is not including

Reply:
kk